Pod/Classes/TSSwiftMarkdownParser.swift

```diff
@@ -98,6 +100,19 @@ public class TSSwiftMarkdownParser: TSBaseParser {
         strongAndEmphasisAttributes = [NSFontAttributeName: strongAndEmphasisFont]
         
         if withDefaultParsing {
+            
+            addNumberedListParsingWithMaxLevel(0, leadFormattingBlock: { (attributedString, range, level) in
```

Comment:
The 0 is hard coded for the `standardParser` which includes all the rules by default.  You can add your own parsing rules and have the max level be bigger than 0.  If that is the case then the line you cited is to fill in the regex on how many levels it should look for the lead string.  The level when not being used here is used on the formattingBlock based on the length of the second match of the regex.  See lines 223-227